test/Templates.Test/Templates.Test.csproj

```diff
@@ -4,7 +4,7 @@
     <DefineConstants>$(DefineConstants);XPLAT</DefineConstants>
     
     <!-- Workaround until https://github.com/aspnet/Common/pull/385 is done -->
-    <DefineConstants>$(DefineConstants);NETCOREAPP2_2</DefineConstants>
+    <DefineConstants>$(DefineConstants);NETCOREAPP3_0</DefineConstants>
```

Comment:
You can remove this altogether now that https://github.com/aspnet/Common/pull/385 is fixed.